src/transactions/instances/transactionPayload.ts

```diff
@@ -511,3 +513,187 @@ export class MultiSigTransactionPayload extends Serializable {
     return new MultiSigTransactionPayload(EntryFunction.deserialize(deserializer));
   }
 }
+
+export enum TransactionInnerPayloadVariants {
```

Comment:
Does this match the rust enum? if so lets add a link to it.
Also, lets add a doc string for this enum

Reply:
Also, can this be places in the `src/types/types.ts` file like all the others https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/types/types.ts#L72

Reply:
Yep, will do.  I got caught up in serialization issues, that I didn't move them to the appropriate locations.